nyan/value/text.cpp

```diff
@@ -79,10 +79,18 @@ const std::unordered_set<nyan_op> &Text::allowed_operations(const Type &with_typ
 		nyan_op::ADD_ASSIGN,
 	};
 
-	if (with_type.get_primitive_type() == primitive_t::TEXT) {
+	const static std::unordered_set<nyan_op> none_ops{
+		nyan_op::ASSIGN,
+	};
+
+	switch (with_type.get_primitive_type()) {
+	case primitive_t::TEXT:
 		return ops;
-	}
-	else {
+
+	case primitive_t::NONE:
+		return none_ops;
```

Comment:
why can we assign `= None` to a text?

Reply:
https://github.com/SFTtech/nyan/pull/84#discussion_r558266843